src/main/java/seedu/bankwithus/AccountList.java

```diff
@@ -1,11 +1,31 @@
 package seedu.bankwithus;
 
+import java.io.IOException;
+import java.util.ArrayList;
 import java.util.Scanner;
 
 public class AccountList {
+    protected ArrayList<Account> accounts;
+    public Ui ui;
```

Comment:
Apparently public attributes need to be declared before private/protected attributes, according to checkstyle. Anyway, ui should be a private attribute, since if another class needs to access ui, the class will just instantiate one.